internal/pkg/manifest/validate.go

```diff
@@ -44,9 +44,132 @@ func (l *LoadBalancedWebServiceConfig) Validate() error {
 	if err = l.PublishConfig.Validate(); err != nil {
 		return fmt.Errorf(`validate "publish": %w`, err)
 	}
-	for i := 0; i < len(l.TaskDefOverrides); i++ {
-		if err = l.TaskDefOverrides[i].Validate(); err != nil {
-			return fmt.Errorf(`validate taskdef_overrides[%d]: %w`, i, err)
+	for ind, taskDefOverride := range l.TaskDefOverrides {
+		if err = taskDefOverride.Validate(); err != nil {
+			return fmt.Errorf(`validate taskdef_overrides[%d]: %w`, ind, err)
+		}
+	}
+	return nil
+}
+
+// Validate returns if BackendServiceConfig is configured correctly.
```

Comment:
nit: these comment (here and below) makes it sound like these methods return bools. Maybe: 
```suggestion
// Validate returns nil if BackendServiceConfig is configured correctly.
```
or some other rewording?

Reply:
good idea!